### DISCUSSION TITLE
how to make a panel (used to be on Community.Toolkit and need to make it on ReactieUI)

### DISCUSSION THREAD

**Discussion by al-zakaria** [AvaloniaUI/Avalonia#14648] [Q&A]
In general, this panel implements its opening and closing, as well as switching pages through this panel, but I need to rewrite it in Reactive, there are literally 2 files with more or less large code, namely MainWindow.axaml and MainWindowViewModel.cs. I couldn't find a solution on the Internet

here is the link to the project

https://github.com/Zakarayaev/avproj4

**Comment by timunie** [comment]
I don't know what exact question do you have to us? And why do you need to move to RxUI? You simply need to change the ViewModelBase to be `ObservableObject` https://github.com/Zakarayaev/avproj4/blob/a669f79a5cc12d2a81192319540817eb64abfa5f/ViewModels/ViewModelBase.cs#L5

```diff
-   public class ViewModelBase : ReactiveObject
+   public class ViewModelBase : ObservableObject
    {
    }
```

**Reply by al-zakaria** [reply]
no, just the code in MainWindowViewModel.cs is code made in Community.Toolkit and I need to change it to Reactive, but I don’t know how

**Reply by timunie** [reply]
well ... https://www.reactiveui.net/docs/getting-started/index.html
